modules/uhttp/handler.go

```diff
@@ -59,5 +59,7 @@ type handlerWrapper struct {
 // ServeHTTP calls Handler.ServeHTTP(ctx, w, r) and injects a new service context for use.
 func (h *handlerWrapper) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	ctx := fxcontext.New(context.Background(), h.host)
+	stopwatch := h.host.Metrics().Timer("http." + r.Method + ".latency").Start()
```

Comment:
Also is "latency" really the right term? This isn't waiting, this is execution time